channels/migrations/0028_blankable_fields.py

```diff
@@ -0,0 +1,42 @@
+# Generated by Django 2.2.27 on 2022-06-09 20:36
+
+from django.db import migrations, models
+import profiles.utils
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [("channels", "0027_post_exclude_from_frontpage_emails")]
+
+    operations = [
+        migrations.AlterField(
+            model_name="channel",
+            name="avatar_medium",
+            field=models.ImageField(
+                blank=True,
+                max_length=2083,
+                null=True,
+                upload_to=profiles.utils.avatar_uri_medium,
+            ),
+        ),
+        migrations.AlterField(
+            model_name="channel",
+            name="avatar_small",
+            field=models.ImageField(
+                blank=True,
+                max_length=2083,
+                null=True,
+                upload_to=profiles.utils.avatar_uri_small,
+            ),
+        ),
+        migrations.AlterField(
+            model_name="channel",
+            name="banner",
+            field=models.ImageField(
+                blank=True,
+                max_length=2083,
+                null=True,
+                upload_to=profiles.utils.banner_uri,
```

Comment:
why  "profiles"?

Reply:
I don't recall, the only thing that really changed in this migration is `blankable=True` instead of `False`, which makes it easier to deal with in django admin.